### PR TITLE
 Revert using types for addons versions for 1.10 branch

### PR DESCRIPTION
**Pull request by moelsayed**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,13 +7,6 @@ TARGETS := $(shell ls scripts)
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
-$(TARGETS): .dapper
```
Comment (deniseschannon) [inline Makefile]: Line 10/11 seem to be in the file before we made changes? https://github.com/rancher/kubernetes-package/pull/138/commits/8998f276c8d8bae1b867439bbb67951909fdc50e#diff-b67911656ef5d18c4ae36cb6741b7965L10

File: package/addons-update.sh [hunk 2]
```diff
@@ -67,21 +67,20 @@ else
...
-source addons_image_list
-# rancher/types has the images with registry namepsace. Since we have that configurable here, we need to remove it:
-TILLER_IMAGE=${TILLER_IMAGE#*/}
-KUBEDNS_IMAGE=${KUBEDNS_IMAGE#*/}
-DNSMASQ_IMAGE=${DNSMASQ_IMAGE#*/}
-GRAFANA_IMAGE=${GRAFANA_IMAGE#*/}
-HEAPSTER_IMAGE=${HEAPSTER_IMAGE#*/}
-INFLUXDB_IMAGE=${INFLUXDB_IMAGE#*/}
-DASHBOARD_IMAGE=${DASHBOARD_IMAGE#*/}
-DNS_SIDECAR_IMAGE=${DNS_SIDECAR_IMAGE#*/}
```
Comment (deniseschannon) [inline package/addons-update.sh:75]: It seems like we're missing removing some line? https://github.com/rancher/kubernetes-package/pull/138/commits/8998f276c8d8bae1b867439bbb67951909fdc50e#diff-347380d9bf27d44c881730a8a2906d3cR72

Comment (moelsayed) [reply package/addons-update.sh:75]: Line you referenced is already removed. There was an additional PR here later so the diff looks a bit different.

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**